tests/kv_test.go

```diff
@@ -0,0 +1,344 @@
+package test
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	xlineapi "github.com/xline-kv/go-xline/api/xline"
+	"github.com/xline-kv/go-xline/client"
+	"github.com/xline-kv/go-xline/xlog"
+	"go.uber.org/zap/zapcore"
+)
+
+func TestKv(t *testing.T) {
+	xlog.SetLevel(zapcore.WarnLevel)
```

Comment:
Debug level could be better in tests.

Reply:
Thank you for the feedback. I've updated it as you recommended.